src/patternfly/components/Scrollspy/scrollspy.scss

```diff
@@ -0,0 +1,168 @@
+  --pf-c-scrollspy--PaddingRight: var(--pf-global--spacer--lg);
+  --pf-c-scrollspy--PaddingLeft: var(--pf-global--spacer--lg);
+
+  // list
+  --pf-c-scrollspy__list--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-scrollspy__list--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-scrollspy__list--before--BorderColor: var(--pf-global--BorderColor--100);
+  --pf-c-scrollspy__list--before--BorderTopWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-scrollspy__list--before--BorderRightWidth: 0;
+  --pf-c-scrollspy__list--before--BorderBottomWidth: 0;
+  --pf-c-scrollspy__list--before--BorderLeftWidth: 0;
+  --pf-c-scrollspy--m-vertical__list--before--BorderLeftWidth: var(--pf-global--BorderWidth--sm);
+  --pf-c-scrollspy--m-vertical__list--before--BorderTopWidth: 0;
+
+  // with background
+  --pf-c-scrollspy--m-background--BackgroundColor: var(--pf-global--BackgroundColor--200);
+
+  // link
+  --pf-c-scrollspy__link--PaddingTop: var(--pf-global--spacer--md);
+  --pf-c-scrollspy__link--PaddingRight: var(--pf-global--spacer--md);
+  --pf-c-scrollspy__link--PaddingBottom: var(--pf-global--spacer--md);
+  --pf-c-scrollspy__link--PaddingLeft: var(--pf-global--spacer--md);
+  --pf-c-scrollspy__link--link--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-scrollspy__link--OutlineOffset: calc(-1 * var(--pf-global--spacer--sm));
+
+  // before
+  --pf-c-scrollspy__link--before--BorderTopWidth: 0;
+  --pf-c-scrollspy__link--before--BorderRightWidth: 0;
+  --pf-c-scrollspy__link--before--BorderBottomWidth: 0;
+  --pf-c-scrollspy__link--before--BorderLeftWidth: 0;
+  --pf-c-scrollspy__link--before--BorderColor: transparent;
+  --pf-c-scrollspy__link--focus--before--BorderTopWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-scrollspy__link--focus--before--BorderLeftWidth: 0;
+  --pf-c-scrollspy__link--focus--before--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-scrollspy__item--m-current__link--before--BorderTopWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-scrollspy__item--m-current__link--before--BorderLeftWidth: 0;
+  --pf-c-scrollspy__item--m-current__link--before--BorderColor: var(--pf-global--primary-color--100);
+  --pf-c-scrollspy--m-vertical__link--focus--before--BorderTopWidth: 0;
+  --pf-c-scrollspy--m-vertical__link--focus--before--BorderLeftWidth: var(--pf-global--BorderWidth--lg);
+  --pf-c-scrollspy--m-vertical__item--m-current__link--before--BorderTopWidth: 0;
+  --pf-c-scrollspy--m-vertical__item--m-current__link--before--BorderLeftWidth: var(--pf-global--BorderWidth--lg);
+
+  // text
+  --pf-c-scrollspy__item-text--Color: var(--pf-global--Color--200);
+  --pf-c-scrollspy__link--hover__item-text--Color: var(--pf-global--Color--100);
+  --pf-c-scrollspy__link--focus__item-text--Color: var(--pf-global--Color--100);
+  --pf-c-scrollspy__item--m-current__item-text--Color: var(--pf-global--Color--100);
+
+  // label
+  --pf-c-scrollspy__label--PaddingTop: var(--pf-global--spacer--md);
```

Comment:
Do you need this? Shouldn't it come from the container it's in?

Reply:
<img width="694" alt="Screen Shot 2020-10-21 at 1 55 12 PM" src="https://user-images.githubusercontent.com/20118816/96758685-12bf7180-13a5-11eb-83d5-33555fbdab79.png">

This is what happens when i move the padding to the element , it shouldnt have this space above it only space above the label when the label is used. Should i move it back?

Reply:
Using the screenshot from https://github.com/patternfly/patternfly/pull/3596#discussion_r509496832, it looks like the padding above the scrollspy label would come from the section it's in, not from the scrollspy component. Typically we don't add outside space like that, and let the space be defined by the spacing of where it's used. That would mean just adding space to the bottom of the label (though I think it should be a margin, not padding)